.circleci/config.yml

```diff
@@ -202,10 +202,17 @@ commands:
       module_target:
         description: module target
         type: string
+      inject_token:
+        description: whether to inject an access token file
+        type: boolean
+        default: false
     steps:
       - run:
           name: Assemble Instrumentation Test APK for << parameters.module_target >>
           command: |
+            if << parameters.inject_token >>; then
+                echo "${MAPBOX_DEVELOPER_CONFIG}" > /root/code/<< parameters.module_target >>/src/androidTest/res/values/mapbox_access_token.xml
```

Comment:
Why do we need this token here?

Reply:
the `app-tests-wrapper` should already carry the token

Reply:
OK, I'll try removing it.

Reply:
Nope, it fails with
```
e: /root/code/libnavigation-core/src/androidTest/java/com/******/navigation/core/trip/service/NativeNavigatorTest.kt: (70, 62): Unresolved reference: ******_access_token
```

Reply:
We already have an `instrumentation-tests` module, which hosts various instrumentation tests and is already configured to have the access token injected.
Can we move `NativeNavigatorCallbackOrderTest.kt` and all `.json` fixtures there and revert changes to this `config.yml` file?